packages/migrations/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@prairielearn/migrations",
+  "version": "1.0.0",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "tsc && copyfiles -u 1 \"./src/**/*.sql\" dist",
```

Comment:
`tsc` doesn't natively support copying non-TS files, see https://github.com/microsoft/TypeScript/issues/30835. This `copyfiles` util was the best option I could find.

Reply:
Yuck, but I agree with this solution.